app/controllers/books_controller.rb

```diff
@@ -12,7 +13,7 @@ def new
   end
 
   def create
-    @book = Book.new(book_params)
+    @book = current_user.books.build(book_params)
```

Comment:
Not sure why we use `build` here instead?
```suggestion
    @book = current_user.books.new(book_params)
```